### PR TITLE
⚗️ Perform experiments - MarkerIO

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

Add markerIO to test capabilities and evaluate

## How to test

This need to be on public network for @mguidon to test the functionality. 

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/boot/index.html [hunk 1]
```diff
@@ -56,4 +56,12 @@
   ${preBootJs}
   <script type="text/javascript" src="${appPath}boot.js"></script>
 </body>
+<script>
+  window.markerConfig = {
+    project: '655285eac2fdc551da905141',
+    source: 'snippet'
+  };
+
```
Comment (pcrespov) [inline services/static-webserver/client/source/boot/index.html:64]: IMO this should only be available in the s4l products

Comment (pcrespov) [reply services/static-webserver/client/source/boot/index.html:64]: Did you get an account?

Comment (mguidon) [reply services/static-webserver/client/source/boot/index.html:64]: I created a test account and just want to see how it works. We well configure it properly (only visible after login, only for sim4life.io) later.

**Review by mguidon — APPROVED** [review]
Thanks

**Review by pcrespov — COMMENTED** [review]
This should not show in osparc-related products

**Review by ignapas — APPROVED** [review]

**Pull request merged**